docs/internals/vsr.md

```diff
@@ -49,8 +49,8 @@ Storage:
 |                `request_reply` | replica |      replica | [Repair Client Replies](#protocol-repair-client-replies), [Sync Client Replies](#protocol-sync-client-replies) |
 |                      `headers` | replica |      replica | [Repair Journal](#protocol-repair-journal)                                       |
 |                     `eviction` | primary |       client | [Client](#protocol-client)                                                       |
-|               `request_blocks` | replica |      replica | [Sync Grid](#protocol-sync-grid), [Repair Grid](#protocol-repair-grid)           |
-|                        `block` | replica |      replica | [Sync Grid](#protocol-sync-grid), [Repair Grid](#protocol-repair-grid)           |
+|               `request_blocks` | replica |      replica | [Sync Tables](#protocol-sync-tables), [Repair Grid](#protocol-repair-grid)       |
+|                        `block` | replica |      replica | [Sync Tables](#protocol-sync-tables), [Repair Grid](#protocol-repair-grid)       |
```

Comment:
Renamed this protocol since it doesn't actually sync manifest blocks, only index/filter/data blocks.